configs/pointpillars/metafile.yml

```diff
@@ -0,0 +1,160 @@
+Collections:
+  - Name: PointPillars
+    Metadata:
+      Training Data:
```

Comment:
we should fill this for each specific model. Otherwise people might think the model is trained on 5 datasets